rust_src/src/indent.rs

```diff
@@ -0,0 +1,21 @@
+use lisp::LispObject;
+use remacs_macros::lisp_fn;
```

Comment:
You forgot to add //! Indentation functions. in the first line.

Reply:
Ah. Ok.